CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ if(DEFINED CMAKE_JS_VERSION)
             set(CMAKE_TOOLCHAIN_FILE "C:\\src\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake")
         endif()
     elseif(APPLE)
-        set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_LIST_DIR}/vendor/realm-core/tools/cmake/macosx.toolchain.cmake")
-        if(DEFINED ENV{NODE_ARCH_ARM})
+      set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9")
```

Comment:
Why was this change made?

Reply:
https://github.com/realm/realm-core/pull/5271